### PR TITLE
New command

### LINKED ISSUE #67: Skeleton generator
Cobalt should provide a basic command for creating a new project with config file and everything setup.

```
cobalt new MyBlog
```

### PR DESCRIPTION
**Pull request by LucioFranco**
This command can be called via `cobalt new myBlog`. There are no tests because I could not find a way to write good tests for this type of thing. This should fix #67.

### CODE REVIEW

**Comment by LucioFranco** [comment]
@johannhof @tak1n do you guys have any objections to this PR and #97?

File: src/main.rs [hunk 1]
```diff
@@ -208,6 +210,25 @@ fn main() {
...
+                match create_new_project(&dest) {
+                    Ok(_) => info!("Created new project at {}", dest),
+                    Err(e) => {
+                        error!("{}", e);
+                        error!("Could not create a new cobalt project");
+                        std::process::exit(1);
+                    }
+                }
+            } else {
+                println!("cobalt new DIRECTORY");
```
Comment (johannhof) [inline src/main.rs]: I think simply printing usage might be enough

Comment (johannhof) [inline src/main.rs]: Or on second thought it's a good idea to print something here, but please make it an actual error message, I didn't really understand it at first. Something like "Please specify a project name".

File: src/new.rs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+use std::fs::{DirBuilder, OpenOptions};
+use std::io::Write;
+use error::Result;
+
+pub fn create_new_project<P: AsRef<Path>>(dest: P) -> Result<()> {
+    let dest = dest.as_ref();
+
+    try!(create_folder(&dest));
+
+    try!(create_file(&dest.join(".coblat.yml"), cobalt_yml));
```
Comment (johannhof) [inline src/new.rs]: typo `coblat.yml`

File: src/new.rs [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        {% else %}
+       	  <title>Cobalt.rs Blog</title>
+        {% endif %}
+
+	    <!-- Latest compiled and minified CSS -->
+     	<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css\" integrity=\"sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7\" crossorigin=\"anonymous\">
+      	<!-- Optional theme -->
+     	<!--<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap-theme.min.css\" integrity=\"sha384-fLW2N01lMqjakBkx3l/M9EahuwpSfeNvV63J5ezn3uZzapT0u7EYsXMjQV+0En5r\" crossorigin=\"anonymous\">-->
+
+      <script src=\"https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js\"></script>
```
Comment (johannhof) [inline src/new.rs]: mmh let's not include all these resources for now. let's instead make a follow up bug that thinks about a good set of default files/resources to include

**Comment by LucioFranco** [comment]
@johannhof all fixed! Thanks for the review. I think I'm ready for the merge!

**Comment by johannhof** [comment]
👍

**Pull request merged**